runtime/common/src/migrations.rs

```diff
@@ -94,7 +118,7 @@ pub struct CommonMigrations<Runtime, Council, Tech>(PhantomData<(Runtime, Counci
 impl<Runtime, Council, Tech> Get<Vec<Box<dyn Migration>>>
 	for CommonMigrations<Runtime, Council, Tech>
 where
-	Runtime: pallet_author_mapping::Config,
+	Runtime: pallet_author_mapping::Config + parachain_staking::Config,
```

Comment:
> I'm wondering about the two different approaches here. Trait bound vs generic type. Seems like you could also use the generic type for this PR right?

Trait bound vs associated type? The migration is a struct generic over `T: parachain_staking::Config` so invoking it requires a type that matches this trait bound (so that is why Runtime is bound to parachain_staking::Config).

We can't make `Runtime` an associated type in `impl Get<Vec<Box<dyn Migration>>> for CommonMigrations<Runtime, Council, Tech>` because the `Get<>` trait definition doesn't have associated types.